Symfony.gitignore

```diff
@@ -14,6 +14,8 @@
 !var/cache/.gitkeep
 !var/logs/.gitkeep
 !var/sessions/.gitkeep
+!var/SymfonyRequirements.php
+/.web-server-pid
```

Comment:
This was added in another PR, and can be cleaned up after the merge conflict is resolved.